decidim-conferences/app/commands/decidim/conferences/admin/create_conference.rb

```diff
@@ -85,6 +89,34 @@ def send_notification
             recipient_ids: conference.followers.pluck(:id)
           )
         end
+
+        def participatory_processes(conference)
+          @participatory_processes ||= conference.participatory_space_sibling_scope(:participatory_processes).where(id: @form.participatory_processes_ids)
+        end
+
+        def link_participatory_processes(conference)
+          conference.link_participatory_spaces_resources(participatory_processes(conference), "included_participatory_processes")
+        end
+
+        def assemblies(conference)
+          @assemblies ||= conference.participatory_space_sibling_scope(:assemblies).where(id: @form.assemblies_ids)
+        end
+
+        def link_assemblies(conference)
+          conference.link_participatory_spaces_resources(assemblies(conference), "included_assemblies")
+        end
+
+        def consultations(conference)
+          @consultations ||= conference.participatory_space_sibling_scope(:consultations)
+                                       .includes(:consultation)
+                                       .where(decidim_consultation_id: @form.consultations_ids)
```

Comment:
I'm not sure about this, since it implies it knows a lot about how consultations work internally 😕

Reply:
when you make `conference.participatory_space_sibling_scope(:consultations)` it returns `questions`. The `Consultations` module says that `Question` is a participatory space  :confused: